doc/use-on-icub-head.md

```diff
@@ -0,0 +1,62 @@
+# Use the robotology-superbuild on the icub-head 
+
+## Introduction 
+One of the main goals of the robotology-superbuild is to simplify downloading and building all the software 
+that is necessary to have on the main control computer contained in the head of the iCub robot, whose hostname 
+is `icub-head` (or `pc104` in older iCub versions). This documentation provides the basic information necessary to 
+use the robotology-superbuild on the computer contained in the head of the iCub robot. 
+
+**Warning: if your iCub robot is not currently using the robotology-superbuild, do not attempt to migrate it
+without coordinating with the iCub support.  For any doubt, please open an issue at https://github.com/robotology/icub-support.**
```

Comment:
https://github.com/robotology/icub-tech-support

Reply:
Thanks, fixed in https://github.com/robotology/robotology-superbuild/commit/8fd2aed342f55f02fe780a38a89995f4939145ac .